lib/lightning_web/live/job_live/inspector_form_component.ex

```diff
@@ -145,6 +145,15 @@ defmodule LightningWeb.JobLive.InspectorFormComponent do
 
           <div class="md:col-span-1">
             <Components.Jobs.credential_select form={f} credentials={@credentials} />
+            <button
+              id="new-credential-launcher"
+              type="button"
+              phx-click={
+                Phoenix.LiveView.JS.push("new-credential", value: @job_params)
```

Comment:
I believe `phx-click="new-credential"` and `phx-value={@job_params}` should have the same effect.

I don' think we need the JS.